app/manifest.json

```diff
@@ -74,7 +74,8 @@
     "49-showlatestdownload": {"description": "DOWNLOADS: Show latest download"},
     "50-capturescreenshot": {"description": "UTILITY: Capture current viewport screenshot"},
     "51-capturefullsizescreenshot": {"description": "UTILITY: Capture full size screenshot (max height is 16,348px due to browser limitation)"},
-    "52-forcecapturefullsizescreenshot": {"description": "UTILITY: Force capture full size screenshot (when window is not scrollable)"}
+    "52-forcecapturefullsizescreenshot": {"description": "UTILITY: Force capture full size screenshot (when window is not scrollable)"},
+    "53-lastusedtab": {"description": "TABS: Switch to last used tab"}
```

Comment:
It's been a while, but IIRC these are sorted alphabetically in the settings screen, meaning if you change this to `05-lastusedtab` then it'll show up right after the ones for next and previous tab which is closer to where users would expect to find it.

Reply:
Will it make a user's config loss if I sort command list? in `chrome://extensions/shortcuts`.

Reply:
I've inserted this command after `14-lasttab`